src/components/Navbar.jsx

```diff
@@ -36,18 +36,15 @@ const useStyles = makeStyles((theme) => ({
 
 // This needs to be a class component because it uses a ref
```

Comment:
Just noting this comment here. Is this no longer true? If so can we clear the comment?

Reply:
I do not believe the comment is true since I didn't find any refs in the file I changed. Maybe the refs were removed previously and the comment left in. Do you also want to clear the other `eslint` comment below?

Reply:
Yeah! Good catch

Reply:
Removed in [4c7681f](https://github.com/unchained-capital/caravan/pull/342/commits/4c7681fe2badd60264d2566255b8660f7e6c4151)